src/main/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionContext.java

```diff
@@ -228,6 +238,47 @@ VP8FrameProjection createVP8FrameProjection(
         return nextVP8FrameProjection;
     }
 
+    private int getMaxSequenceNumberOfFrame(@NotNull RawPacket rtpPacket)
```

Comment:
so this is:
```
given an RTP packet, lookup the frame to which it belongs and then find the highest sequence number we've received from that frame
```
?
adding that as a comment would be helpful i think.  also, i'm in favor of passing only what is needed (just the ssrc and timestamp of the packet in this case) as it reduces the 'footprint' of what this method really needs, but don't have to change that.